tests/HomeControllerTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests;
+
+use Tempest\Framework\Testing\IntegrationTest;
+
+class HomeControllerTest extends IntegrationTest
+{
+    protected string $root = __DIR__ . '/../';
```

Comment:
This is needed in order for the TestCase to find `composer.json`. Would be nice if this was not required

Reply:
Hmm yeah, the way I work around that in `tempest/framework` is by implementing an abstract `FrameworkIntegrationTestCase`, which all other tests extend from. That in turn extends from `IntegrationTest`. 

I'm not opposed to having a `ProjectIntegrationTestCase` as well, shipped with `tempest/app`. WDYT?

For reference: https://github.com/tempestphp/tempest-framework/blob/main/tests/Integration/FrameworkIntegrationTestCase.php

Reply:
I think its a good idea to ship an `IntegrationTestCase`. I'm not sure about what would be the best name for it. I've used `IntegrationTestCase` for now, but open to other suggestions.